docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+It holds information such as name, remaining money, and history of transaction.
+The DApp uses that record to manage the user while using the application.
+
+For example, a user's account balance would help the DApp to decide if the user is eligible to complete a transaction or not
+:::
+
+Just before the contracts, type the following code:
+
+```reach
+load: /examples/mkt-2-basic/index.mjs
+range: 5 - 8
+```
+
+* Line 5 sets up some funds that can be used by each `{!rsh} Participant`s as starting balance.
+`{!rsh} parseCurrency` converts the figure passed into it to an acceptable currency on a DApp.
+
+:::note
+`{!rsh} parseCurrency` `{!rsh} function` is an in-built method for converting regular currencies into currencies acceptable on a DApp.
+:::
+
+* Lines 7 and 8 create an account for each of the `{!rsh} Participant`.
+This is for test purposes. 
+
+With these accounts, the contracts will compile successfully when called upon.
+
+### {#tut-mkt-basic-define} Define Variables
+Having told the backend how those variables declared are to be used, these variables will now be given their proper functionalities.
+
+Just below the contracts, enter the following code for the `sellerInteract`:
+
+```reach
+load: /examples/mkt-2-basic/index.mjs
+range: 13 - 30
+```
+
+This code will be made accessible to the `Seller` only.
+
+* Line 13 defines `sellerInteract` as an `{!rsh} Object`.
+
+* Lines 14 to 21 defines the `sellerInfo` as an `{!rsh} Object`.
+
+* Line 15 passes a `{!rsh} string` to the `announcement` variable.
+
+* Lines 16 to 20: The details of each `product` is passed into the `products` `{!rsh} Array` as `{!rsh} Object`s.
+It is a list of items that the `Seller` puts up for sale with their properties.
+
+* Lines 22 to 29 creates a  `{!rsh} function` named `reportReady`. 
+The `{!rsh} function` takes in `announcement` and `products` as arguments.
+
+* Line 23 logs a `{!rsh} string` to the console.
+
+* Lines 24 to 28 obtain the contract information which is an `{!rsh} Object` and log it to the console as a JSON string.
+You can see this as the seller's unique identifier.
+
+That is all that the `Seller` needs for this section.
+
+
+The next code is for the `buyerInteract`:
+
+```reach
+load: /examples/mkt-2-basic/index.mjs
+range: 32 - 47
+```
+
+The `Buyer` will have access to this code.
+
+* Line 32 defines `buyerInteract` as an `{!rsh} Object`.
+
+* Line 33 defines `shop` as a `{!rsh} function`. It takes the `sellerInfo` as an argument.
+
+* Line 34 logs the `Seller`'s `announcement` to the console.
+
+* Lines 35 to 39 loop through the `Seller`'s `products` and list them out for the `Buyer` to see.
+
+* Line 41 gets a random number not more than `4`.
+This number stored as `choice` represents the position of an item in the `products` array.
+So if the `choice` is `2`, then the `product` chosen will be 
+`{ name: "Corn", unit: "ear", units: "ears", price: "5" }`
+
+* Line 41 picks the `quantity` of that `product` the `Buyer` wants.
```

Comment:
Line 41 picks the `quantity` of that `product` for the `Buyer`.

Since these are just running tests, it is not what the `Buyer` wants, but here is a good place to put in some information about how this would be used in a deployed DApp. "In a DApp deployed on the mainnet, a buyer would select blah, blah, blah, but in this tutorial, we blah blah blah"